src/options.js

```diff
@@ -14,37 +14,19 @@
  * You should have received a copy of the GNU General Public License
  * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/>.
  */
+ // TODO: This code is a hideous mess and desperately needs to be refactored and cleaned up.
```

Comment:
I mean, it's a hideous mess because popup.js was a hideous mess. Not because of anything you did :smile_cat: